yami-ichi.md

```diff
@@ -2,13 +2,28 @@
 layout: page
 order: 2
 section: "2018"
-title: "Kickoff Yami-ichi"
-titleDisplay: "Yami-Ichi"
+title: "Kickoff Party: Bit Bazaar × Internet Yami-ichi"
+titleDisplay: "Bit Bazaar × Internet Yami-Ichi"
```

Comment:
Seems a bit long relative to the other items. Would it work if it were `Yami-Ichi` in the menu and the full title on the page?

Reply:
Oh yes - that makes sense.  I didn't realize one was the link, thought they were both for the page somehow

Reply:
+1 short title yeah that is my poor wording in frontmatter!